examples/tutorials/translations/bengali/Part 06 - Federated Learning on MNIST using a CNN.ipynb

```diff
@@ -0,0 +1,367 @@
+{
```

Comment:
<p>Fix markdown link syntax for blog link</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/3052/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>